src/main/kotlin/com/cognifide/gradle/aem/pkg/deploy/InstallResponse.kt

```diff
@@ -1,35 +1,62 @@
 package com.cognifide.gradle.aem.pkg.deploy
 
+import java.io.InputStream
 import java.util.regex.Pattern
 
-class InstallResponse(private val rawHtml: String) : HtmlResponse(rawHtml) {
+class InstallResponse private constructor(private val rawHtml: String, private val packageErrors: Set<String>) : HtmlResponse(rawHtml) {
```

Comment:
`packageErrors` is not being a part `InstallResponse` semantically so that contract here is not well designed for me.

this `packageErrors` should not be here or at least be an arg of method `fun hasPackageErrors(errors: Collection<String>)` // collection here is intentional here because I am accepting the weakest contract possible to avoid extra list to set conversions which costs performace/memory